post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -12,21 +12,29 @@ The Multiarch Tuning Operator is a Technology Preview feature only. Technology P
 For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
 
-The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration. 
+The Multiarch Tuning Operator supports architecture-aware workload scheduling on multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
```

Comment:
Usage of "operational experience" was one of the biggest concerns that the peer reviewers had.

And it is because it makes it sound more like "marketing content" which also makes it vague. We must try and be as specific as possible. In the future, if the Operator does many other things, we can definitely expand this sentence to include all of those actions.

But even for now, I would highly suggest that we keep it specific rather than generic.
WDYT? @aleskandro

Reply:
I don't think it's 'marketing content'. Specifically, the operator is meant to support a better operational experience. One of its operands supports architecture-aware scheduling. I'm not sure how to address the concern, but not by having the concept we want to communicate about this project since the beginning is not the one proposed in the above change.